src/components/VmDetails/BaseCard.js

```diff
@@ -57,6 +57,11 @@ class BaseCard extends React.Component {
     }
   }
 
+  renderChildren = (childProps) => {
```

Comment:
I just posted #735 to combine the `renderChildren` technique while converting the event handler style to match the rest of the code base.